mandatory/4-tax.js

```diff
@@ -5,7 +5,15 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return (price*20/100)
```

Comment:
There is a little misunderstanding because of function name.  
Comment in exercise: "Remember that the prices must include the sales tax (hint: you already wrote a function for this!)"
So Function must calculate price with tax so It have to be like "return (price*120/100)"
Others look great.